### DISCUSSION TITLE
Unpublish GitHub Copilot Completions and GitHub Copilot Chat (the Visual Studio one) from VS Marketplace

### DISCUSSION THREAD

**Discussion by ChaseKnowlden** [community/community#127758] [Copilot Conversations]
### Select Topic Area

General

### Body

These two extensions are built in to Visual Studio 17.10.

**Comment by ayim** [comment]
Hi @ChaseKnowlden,

Great observation.
I know that this can be confusing and we're updating the marketplace extension text to make the distinction more clear to users.
The extensions still need to be published in the marketplace for 17.8 and 17.9 users.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
aayim@microsoft.com
